report/src/main/java/com/hartwig/actin/report/pdf/util/Cells.java

```diff
@@ -118,6 +118,13 @@ public static Cell createContent(@NotNull String text) {
         return createContent(new Paragraph(text));
     }
 
+    @NotNull
+    public static Cell createContentGrey(@NotNull String text) {
+        Cell cell = createContent(text);
+        cell.setFontColor(Styles.PALETTE_MID_GREY);
+        return cell;
+    }
+
     @NotNull
```

Comment:
Still don't like this name. All functions in Cells are named after their "functional usage". Eg "createContentWarn" could have been called "createContentOrange" but then users wouldn't know when to use this function consistently.

I think rather than use createContentGrey I'd go for something like createContentDeamphasize, or perhaps there is a better word to describe what we try to achieve with greying out certain content?

Reply:
That makes sense. I can fix this in #12.